Nov 12 11:24:35 ravinder-adsb frpc[670]: 2023/11/12 11:24:35 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:35 ravinder-adsb frpc[670]: 2023/11/12 11:24:35 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:37 ravinder-adsb frpc[670]: 2023/11/12 11:24:37 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:37 ravinder-adsb frpc[670]: 2023/11/12 11:24:37 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:39 ravinder-adsb frpc[670]: 2023/11/12 11:24:39 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:39 ravinder-adsb frpc[670]: 2023/11/12 11:24:39 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:41 ravinder-adsb frpc[670]: 2023/11/12 11:24:41 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:41 ravinder-adsb frpc[670]: 2023/11/12 11:24:41 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:43 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:24:43 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:24:43 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 347ms
Nov 12 11:24:43 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:24:43 ravinder-adsb frpc[670]: 2023/11/12 11:24:43 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:43 ravinder-adsb frpc[670]: 2023/11/12 11:24:43 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:45 ravinder-adsb frpc[670]: 2023/11/12 11:24:45 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:45 ravinder-adsb frpc[670]: 2023/11/12 11:24:45 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:47 ravinder-adsb frpc[670]: 2023/11/12 11:24:47 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:47 ravinder-adsb frpc[670]: 2023/11/12 11:24:47 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:48 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:24:49 ravinder-adsb frpc[670]: 2023/11/12 11:24:49 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:49 ravinder-adsb frpc[670]: 2023/11/12 11:24:49 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:51 ravinder-adsb frpc[670]: 2023/11/12 11:24:51 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:51 ravinder-adsb frpc[670]: 2023/11/12 11:24:51 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:53 ravinder-adsb frpc[670]: 2023/11/12 11:24:53 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:53 ravinder-adsb frpc[670]: 2023/11/12 11:24:53 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:55 ravinder-adsb frpc[670]: 2023/11/12 11:24:55 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:55 ravinder-adsb frpc[670]: 2023/11/12 11:24:55 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:57 ravinder-adsb frpc[670]: 2023/11/12 11:24:57 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:57 ravinder-adsb frpc[670]: 2023/11/12 11:24:57 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:24:58 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:24:58 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:24:58 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 347ms
Nov 12 11:24:58 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:24:59 ravinder-adsb frpc[670]: 2023/11/12 11:24:59 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:24:59 ravinder-adsb frpc[670]: 2023/11/12 11:24:59 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:01 ravinder-adsb frpc[670]: 2023/11/12 11:25:01 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:01 ravinder-adsb frpc[670]: 2023/11/12 11:25:01 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:03 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:25:03 ravinder-adsb frpc[670]: 2023/11/12 11:25:03 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:03 ravinder-adsb frpc[670]: 2023/11/12 11:25:03 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:05 ravinder-adsb frpc[670]: 2023/11/12 11:25:05 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:05 ravinder-adsb frpc[670]: 2023/11/12 11:25:05 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:07 ravinder-adsb frpc[670]: 2023/11/12 11:25:07 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:07 ravinder-adsb frpc[670]: 2023/11/12 11:25:07 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:09 ravinder-adsb frpc[670]: 2023/11/12 11:25:09 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:09 ravinder-adsb frpc[670]: 2023/11/12 11:25:09 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:11 ravinder-adsb frpc[670]: 2023/11/12 11:25:11 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:11 ravinder-adsb frpc[670]: 2023/11/12 11:25:11 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:13 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:25:13 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:25:13 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 351ms
Nov 12 11:25:13 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:25:13 ravinder-adsb frpc[670]: 2023/11/12 11:25:13 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:13 ravinder-adsb frpc[670]: 2023/11/12 11:25:13 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:15 ravinder-adsb frpc[670]: 2023/11/12 11:25:15 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:15 ravinder-adsb frpc[670]: 2023/11/12 11:25:15 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:17 ravinder-adsb frpc[670]: 2023/11/12 11:25:17 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:17 ravinder-adsb frpc[670]: 2023/11/12 11:25:17 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:18 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:25:19 ravinder-adsb frpc[670]: 2023/11/12 11:25:19 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:19 ravinder-adsb frpc[670]: 2023/11/12 11:25:19 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:21 ravinder-adsb frpc[670]: 2023/11/12 11:25:21 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:21 ravinder-adsb frpc[670]: 2023/11/12 11:25:21 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:23 ravinder-adsb frpc[670]: 2023/11/12 11:25:23 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:23 ravinder-adsb frpc[670]: 2023/11/12 11:25:23 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:25 ravinder-adsb frpc[670]: 2023/11/12 11:25:25 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:25 ravinder-adsb frpc[670]: 2023/11/12 11:25:25 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:27 ravinder-adsb frpc[670]: 2023/11/12 11:25:27 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:27 ravinder-adsb frpc[670]: 2023/11/12 11:25:27 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:28 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:25:28 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:25:28 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 356ms
Nov 12 11:25:28 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:25:29 ravinder-adsb frpc[670]: 2023/11/12 11:25:29 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:29 ravinder-adsb frpc[670]: 2023/11/12 11:25:29 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:31 ravinder-adsb frpc[670]: 2023/11/12 11:25:31 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:31 ravinder-adsb frpc[670]: 2023/11/12 11:25:31 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:33 ravinder-adsb frpc[670]: 2023/11/12 11:25:33 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:33 ravinder-adsb frpc[670]: 2023/11/12 11:25:33 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:33 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:25:35 ravinder-adsb frpc[670]: 2023/11/12 11:25:35 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:35 ravinder-adsb frpc[670]: 2023/11/12 11:25:35 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:37 ravinder-adsb frpc[670]: 2023/11/12 11:25:37 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:37 ravinder-adsb frpc[670]: 2023/11/12 11:25:37 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:39 ravinder-adsb frpc[670]: 2023/11/12 11:25:39 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:39 ravinder-adsb frpc[670]: 2023/11/12 11:25:39 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:41 ravinder-adsb frpc[670]: 2023/11/12 11:25:41 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:41 ravinder-adsb frpc[670]: 2023/11/12 11:25:41 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:43 ravinder-adsb frpc[670]: 2023/11/12 11:25:43 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:43 ravinder-adsb frpc[670]: 2023/11/12 11:25:43 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:43 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:25:43 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:25:43 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 356ms
Nov 12 11:25:43 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:25:45 ravinder-adsb frpc[670]: 2023/11/12 11:25:45 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:45 ravinder-adsb frpc[670]: 2023/11/12 11:25:45 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:47 ravinder-adsb frpc[670]: 2023/11/12 11:25:47 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:47 ravinder-adsb frpc[670]: 2023/11/12 11:25:47 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:48 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:25:49 ravinder-adsb frpc[670]: 2023/11/12 11:25:49 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:49 ravinder-adsb frpc[670]: 2023/11/12 11:25:49 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:51 ravinder-adsb frpc[670]: 2023/11/12 11:25:51 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:51 ravinder-adsb frpc[670]: 2023/11/12 11:25:51 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:53 ravinder-adsb frpc[670]: 2023/11/12 11:25:53 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:53 ravinder-adsb frpc[670]: 2023/11/12 11:25:53 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:55 ravinder-adsb frpc[670]: 2023/11/12 11:25:55 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:55 ravinder-adsb frpc[670]: 2023/11/12 11:25:55 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:57 ravinder-adsb frpc[670]: 2023/11/12 11:25:57 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:57 ravinder-adsb frpc[670]: 2023/11/12 11:25:57 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:25:58 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:25:58 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:25:58 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 358ms
Nov 12 11:25:58 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:25:59 ravinder-adsb frpc[670]: 2023/11/12 11:25:59 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:25:59 ravinder-adsb frpc[670]: 2023/11/12 11:25:59 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:01 ravinder-adsb frpc[670]: 2023/11/12 11:26:01 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:01 ravinder-adsb frpc[670]: 2023/11/12 11:26:01 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:03 ravinder-adsb frpc[670]: 2023/11/12 11:26:03 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:03 ravinder-adsb frpc[670]: 2023/11/12 11:26:03 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:03 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:26:05 ravinder-adsb frpc[670]: 2023/11/12 11:26:05 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:05 ravinder-adsb frpc[670]: 2023/11/12 11:26:05 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:07 ravinder-adsb frpc[670]: 2023/11/12 11:26:07 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:07 ravinder-adsb frpc[670]: 2023/11/12 11:26:07 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:09 ravinder-adsb frpc[670]: 2023/11/12 11:26:09 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:09 ravinder-adsb frpc[670]: 2023/11/12 11:26:09 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:11 ravinder-adsb frpc[670]: 2023/11/12 11:26:11 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:11 ravinder-adsb frpc[670]: 2023/11/12 11:26:11 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:13 ravinder-adsb frpc[670]: 2023/11/12 11:26:13 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:13 ravinder-adsb frpc[670]: 2023/11/12 11:26:13 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:13 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:26:13 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:26:13 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 359ms
Nov 12 11:26:13 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:26:15 ravinder-adsb frpc[670]: 2023/11/12 11:26:15 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:15 ravinder-adsb frpc[670]: 2023/11/12 11:26:15 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:17 ravinder-adsb frpc[670]: 2023/11/12 11:26:17 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:17 ravinder-adsb frpc[670]: 2023/11/12 11:26:17 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:18 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:26:19 ravinder-adsb frpc[670]: 2023/11/12 11:26:19 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:19 ravinder-adsb frpc[670]: 2023/11/12 11:26:19 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:21 ravinder-adsb frpc[670]: 2023/11/12 11:26:21 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:21 ravinder-adsb frpc[670]: 2023/11/12 11:26:21 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:23 ravinder-adsb frpc[670]: 2023/11/12 11:26:23 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:23 ravinder-adsb frpc[670]: 2023/11/12 11:26:23 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:25 ravinder-adsb frpc[670]: 2023/11/12 11:26:25 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:25 ravinder-adsb frpc[670]: 2023/11/12 11:26:25 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:27 ravinder-adsb frpc[670]: 2023/11/12 11:26:27 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:27 ravinder-adsb frpc[670]: 2023/11/12 11:26:27 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:28 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:26:28 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:26:28 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 361ms
Nov 12 11:26:28 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:26:29 ravinder-adsb frpc[670]: 2023/11/12 11:26:29 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:29 ravinder-adsb frpc[670]: 2023/11/12 11:26:29 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:31 ravinder-adsb frpc[670]: 2023/11/12 11:26:31 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:31 ravinder-adsb frpc[670]: 2023/11/12 11:26:31 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:33 ravinder-adsb frpc[670]: 2023/11/12 11:26:33 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:33 ravinder-adsb frpc[670]: 2023/11/12 11:26:33 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:33 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:26:35 ravinder-adsb frpc[670]: 2023/11/12 11:26:35 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:35 ravinder-adsb frpc[670]: 2023/11/12 11:26:35 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:37 ravinder-adsb frpc[670]: 2023/11/12 11:26:37 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:37 ravinder-adsb frpc[670]: 2023/11/12 11:26:37 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:39 ravinder-adsb frpc[670]: 2023/11/12 11:26:39 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:39 ravinder-adsb frpc[670]: 2023/11/12 11:26:39 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:41 ravinder-adsb frpc[670]: 2023/11/12 11:26:41 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:41 ravinder-adsb frpc[670]: 2023/11/12 11:26:41 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:43 ravinder-adsb frpc[670]: 2023/11/12 11:26:43 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:43 ravinder-adsb frpc[670]: 2023/11/12 11:26:43 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:43 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:26:43 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:26:43 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 363ms
Nov 12 11:26:43 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:26:45 ravinder-adsb frpc[670]: 2023/11/12 11:26:45 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:45 ravinder-adsb frpc[670]: 2023/11/12 11:26:45 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:47 ravinder-adsb frpc[670]: 2023/11/12 11:26:47 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:47 ravinder-adsb frpc[670]: 2023/11/12 11:26:47 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:48 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:26:49 ravinder-adsb frpc[670]: 2023/11/12 11:26:49 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:49 ravinder-adsb frpc[670]: 2023/11/12 11:26:49 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:51 ravinder-adsb frpc[670]: 2023/11/12 11:26:51 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:51 ravinder-adsb frpc[670]: 2023/11/12 11:26:51 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:53 ravinder-adsb frpc[670]: 2023/11/12 11:26:53 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:53 ravinder-adsb frpc[670]: 2023/11/12 11:26:53 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:55 ravinder-adsb frpc[670]: 2023/11/12 11:26:55 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:55 ravinder-adsb frpc[670]: 2023/11/12 11:26:55 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:57 ravinder-adsb frpc[670]: 2023/11/12 11:26:57 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:57 ravinder-adsb frpc[670]: 2023/11/12 11:26:57 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:26:58 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:26:58 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:26:58 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 374ms
Nov 12 11:26:58 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:26:59 ravinder-adsb frpc[670]: 2023/11/12 11:26:59 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:26:59 ravinder-adsb frpc[670]: 2023/11/12 11:26:59 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:01 ravinder-adsb frpc[670]: 2023/11/12 11:27:01 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:01 ravinder-adsb frpc[670]: 2023/11/12 11:27:01 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:03 ravinder-adsb frpc[670]: 2023/11/12 11:27:03 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:03 ravinder-adsb frpc[670]: 2023/11/12 11:27:03 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:03 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:27:05 ravinder-adsb frpc[670]: 2023/11/12 11:27:05 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:05 ravinder-adsb frpc[670]: 2023/11/12 11:27:05 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:07 ravinder-adsb frpc[670]: 2023/11/12 11:27:07 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:07 ravinder-adsb frpc[670]: 2023/11/12 11:27:07 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:09 ravinder-adsb frpc[670]: 2023/11/12 11:27:09 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:09 ravinder-adsb frpc[670]: 2023/11/12 11:27:09 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:11 ravinder-adsb frpc[670]: 2023/11/12 11:27:11 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:11 ravinder-adsb frpc[670]: 2023/11/12 11:27:11 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:13 ravinder-adsb frpc[670]: 2023/11/12 11:27:13 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:13 ravinder-adsb frpc[670]: 2023/11/12 11:27:13 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:13 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:27:13 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:27:13 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 365ms
Nov 12 11:27:13 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:27:15 ravinder-adsb frpc[670]: 2023/11/12 11:27:15 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:15 ravinder-adsb frpc[670]: 2023/11/12 11:27:15 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:17 ravinder-adsb frpc[670]: 2023/11/12 11:27:17 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:17 ravinder-adsb frpc[670]: 2023/11/12 11:27:17 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:18 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:27:19 ravinder-adsb frpc[670]: 2023/11/12 11:27:19 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:19 ravinder-adsb frpc[670]: 2023/11/12 11:27:19 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:21 ravinder-adsb frpc[670]: 2023/11/12 11:27:21 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:21 ravinder-adsb frpc[670]: 2023/11/12 11:27:21 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:23 ravinder-adsb frpc[670]: 2023/11/12 11:27:23 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:23 ravinder-adsb frpc[670]: 2023/11/12 11:27:23 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:25 ravinder-adsb frpc[670]: 2023/11/12 11:27:25 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:25 ravinder-adsb frpc[670]: 2023/11/12 11:27:25 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:27 ravinder-adsb frpc[670]: 2023/11/12 11:27:27 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:27 ravinder-adsb frpc[670]: 2023/11/12 11:27:27 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:28 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:27:28 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:27:28 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 366ms
Nov 12 11:27:28 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:27:29 ravinder-adsb frpc[670]: 2023/11/12 11:27:29 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:29 ravinder-adsb frpc[670]: 2023/11/12 11:27:29 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:31 ravinder-adsb frpc[670]: 2023/11/12 11:27:31 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:31 ravinder-adsb frpc[670]: 2023/11/12 11:27:31 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:33 ravinder-adsb frpc[670]: 2023/11/12 11:27:33 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:33 ravinder-adsb frpc[670]: 2023/11/12 11:27:33 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:33 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:27:35 ravinder-adsb frpc[670]: 2023/11/12 11:27:35 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:35 ravinder-adsb frpc[670]: 2023/11/12 11:27:35 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:37 ravinder-adsb frpc[670]: 2023/11/12 11:27:37 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:37 ravinder-adsb frpc[670]: 2023/11/12 11:27:37 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:39 ravinder-adsb frpc[670]: 2023/11/12 11:27:39 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:39 ravinder-adsb frpc[670]: 2023/11/12 11:27:39 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:41 ravinder-adsb frpc[670]: 2023/11/12 11:27:41 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:41 ravinder-adsb frpc[670]: 2023/11/12 11:27:41 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:43 ravinder-adsb frpc[670]: 2023/11/12 11:27:43 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:43 ravinder-adsb frpc[670]: 2023/11/12 11:27:43 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:43 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:27:43 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:27:43 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 366ms
Nov 12 11:27:43 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:27:45 ravinder-adsb frpc[670]: 2023/11/12 11:27:45 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:45 ravinder-adsb frpc[670]: 2023/11/12 11:27:45 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:47 ravinder-adsb frpc[670]: 2023/11/12 11:27:47 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:47 ravinder-adsb frpc[670]: 2023/11/12 11:27:47 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:49 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:27:49 ravinder-adsb frpc[670]: 2023/11/12 11:27:49 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:49 ravinder-adsb frpc[670]: 2023/11/12 11:27:49 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:51 ravinder-adsb frpc[670]: 2023/11/12 11:27:51 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:51 ravinder-adsb frpc[670]: 2023/11/12 11:27:51 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:53 ravinder-adsb frpc[670]: 2023/11/12 11:27:53 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:53 ravinder-adsb frpc[670]: 2023/11/12 11:27:53 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:55 ravinder-adsb frpc[670]: 2023/11/12 11:27:55 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:55 ravinder-adsb frpc[670]: 2023/11/12 11:27:55 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:57 ravinder-adsb frpc[670]: 2023/11/12 11:27:57 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:57 ravinder-adsb frpc[670]: 2023/11/12 11:27:57 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:27:59 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:27:59 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:27:59 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 368ms
Nov 12 11:27:59 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:27:59 ravinder-adsb frpc[670]: 2023/11/12 11:27:59 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:27:59 ravinder-adsb frpc[670]: 2023/11/12 11:27:59 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:01 ravinder-adsb frpc[670]: 2023/11/12 11:28:01 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:01 ravinder-adsb frpc[670]: 2023/11/12 11:28:01 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:03 ravinder-adsb frpc[670]: 2023/11/12 11:28:03 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:03 ravinder-adsb frpc[670]: 2023/11/12 11:28:03 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:04 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:28:05 ravinder-adsb frpc[670]: 2023/11/12 11:28:05 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:05 ravinder-adsb frpc[670]: 2023/11/12 11:28:05 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:07 ravinder-adsb frpc[670]: 2023/11/12 11:28:07 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:07 ravinder-adsb frpc[670]: 2023/11/12 11:28:07 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:09 ravinder-adsb frpc[670]: 2023/11/12 11:28:09 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:09 ravinder-adsb frpc[670]: 2023/11/12 11:28:09 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:11 ravinder-adsb frpc[670]: 2023/11/12 11:28:11 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:11 ravinder-adsb frpc[670]: 2023/11/12 11:28:11 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:13 ravinder-adsb frpc[670]: 2023/11/12 11:28:13 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:13 ravinder-adsb frpc[670]: 2023/11/12 11:28:13 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:14 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:28:14 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:28:14 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 372ms
Nov 12 11:28:14 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:28:15 ravinder-adsb frpc[670]: 2023/11/12 11:28:15 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:15 ravinder-adsb frpc[670]: 2023/11/12 11:28:15 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:17 ravinder-adsb frpc[670]: 2023/11/12 11:28:17 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:17 ravinder-adsb frpc[670]: 2023/11/12 11:28:17 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:19 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:28:19 ravinder-adsb frpc[670]: 2023/11/12 11:28:19 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:19 ravinder-adsb frpc[670]: 2023/11/12 11:28:19 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:21 ravinder-adsb frpc[670]: 2023/11/12 11:28:21 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:21 ravinder-adsb frpc[670]: 2023/11/12 11:28:21 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:23 ravinder-adsb frpc[670]: 2023/11/12 11:28:23 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:23 ravinder-adsb frpc[670]: 2023/11/12 11:28:23 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:25 ravinder-adsb frpc[670]: 2023/11/12 11:28:25 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:25 ravinder-adsb frpc[670]: 2023/11/12 11:28:25 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:27 ravinder-adsb frpc[670]: 2023/11/12 11:28:27 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:27 ravinder-adsb frpc[670]: 2023/11/12 11:28:27 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:29 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:28:29 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:28:29 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 376ms
Nov 12 11:28:29 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:28:29 ravinder-adsb frpc[670]: 2023/11/12 11:28:29 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:29 ravinder-adsb frpc[670]: 2023/11/12 11:28:29 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:31 ravinder-adsb frpc[670]: 2023/11/12 11:28:31 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:31 ravinder-adsb frpc[670]: 2023/11/12 11:28:31 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:33 ravinder-adsb frpc[670]: 2023/11/12 11:28:33 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:33 ravinder-adsb frpc[670]: 2023/11/12 11:28:33 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:34 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:28:35 ravinder-adsb frpc[670]: 2023/11/12 11:28:35 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:35 ravinder-adsb frpc[670]: 2023/11/12 11:28:35 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:37 ravinder-adsb frpc[670]: 2023/11/12 11:28:37 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:37 ravinder-adsb frpc[670]: 2023/11/12 11:28:37 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:39 ravinder-adsb frpc[670]: 2023/11/12 11:28:39 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:39 ravinder-adsb frpc[670]: 2023/11/12 11:28:39 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:41 ravinder-adsb frpc[670]: 2023/11/12 11:28:41 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:41 ravinder-adsb frpc[670]: 2023/11/12 11:28:41 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:43 ravinder-adsb frpc[670]: 2023/11/12 11:28:43 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:43 ravinder-adsb frpc[670]: 2023/11/12 11:28:43 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:44 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:28:44 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:28:44 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 377ms
Nov 12 11:28:44 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:28:45 ravinder-adsb frpc[670]: 2023/11/12 11:28:45 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:45 ravinder-adsb frpc[670]: 2023/11/12 11:28:45 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:47 ravinder-adsb frpc[670]: 2023/11/12 11:28:47 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:47 ravinder-adsb frpc[670]: 2023/11/12 11:28:47 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:49 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:28:49 ravinder-adsb frpc[670]: 2023/11/12 11:28:49 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:49 ravinder-adsb frpc[670]: 2023/11/12 11:28:49 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:51 ravinder-adsb frpc[670]: 2023/11/12 11:28:51 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:51 ravinder-adsb frpc[670]: 2023/11/12 11:28:51 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:53 ravinder-adsb frpc[670]: 2023/11/12 11:28:53 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:53 ravinder-adsb frpc[670]: 2023/11/12 11:28:53 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:55 ravinder-adsb frpc[670]: 2023/11/12 11:28:55 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:55 ravinder-adsb frpc[670]: 2023/11/12 11:28:55 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:57 ravinder-adsb frpc[670]: 2023/11/12 11:28:57 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:57 ravinder-adsb frpc[670]: 2023/11/12 11:28:57 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:28:59 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:28:59 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:28:59 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 378ms
Nov 12 11:28:59 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:28:59 ravinder-adsb frpc[670]: 2023/11/12 11:28:59 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:28:59 ravinder-adsb frpc[670]: 2023/11/12 11:28:59 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:01 ravinder-adsb frpc[670]: 2023/11/12 11:29:01 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:01 ravinder-adsb frpc[670]: 2023/11/12 11:29:01 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:03 ravinder-adsb frpc[670]: 2023/11/12 11:29:03 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:03 ravinder-adsb frpc[670]: 2023/11/12 11:29:03 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:04 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:29:05 ravinder-adsb frpc[670]: 2023/11/12 11:29:05 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:05 ravinder-adsb frpc[670]: 2023/11/12 11:29:05 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:07 ravinder-adsb frpc[670]: 2023/11/12 11:29:07 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:07 ravinder-adsb frpc[670]: 2023/11/12 11:29:07 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:09 ravinder-adsb frpc[670]: 2023/11/12 11:29:09 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:09 ravinder-adsb frpc[670]: 2023/11/12 11:29:09 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:11 ravinder-adsb frpc[670]: 2023/11/12 11:29:11 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:11 ravinder-adsb frpc[670]: 2023/11/12 11:29:11 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:12 ravinder-adsb python3[23415]: Traceback (most recent call last):
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/urllib/request.py", line 1324, in do_open
Nov 12 11:29:12 ravinder-adsb python3[23415]:     encode_chunked=req.has_header('Transfer-encoding'))
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/http/client.py", line 1260, in request
Nov 12 11:29:12 ravinder-adsb python3[23415]:     self._send_request(method, url, body, headers, encode_chunked)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/http/client.py", line 1306, in _send_request
Nov 12 11:29:12 ravinder-adsb python3[23415]:     self.endheaders(body, encode_chunked=encode_chunked)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/http/client.py", line 1255, in endheaders
Nov 12 11:29:12 ravinder-adsb python3[23415]:     self._send_output(message_body, encode_chunked=encode_chunked)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/http/client.py", line 1030, in _send_output
Nov 12 11:29:12 ravinder-adsb python3[23415]:     self.send(msg)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/http/client.py", line 970, in send
Nov 12 11:29:12 ravinder-adsb python3[23415]:     self.connect()
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/http/client.py", line 1415, in connect
Nov 12 11:29:12 ravinder-adsb python3[23415]:     super().connect()
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/http/client.py", line 942, in connect
Nov 12 11:29:12 ravinder-adsb python3[23415]:     (self.host,self.port), self.timeout, self.source_address)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/socket.py", line 707, in create_connection
Nov 12 11:29:12 ravinder-adsb python3[23415]:     for res in getaddrinfo(host, port, 0, SOCK_STREAM):
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/socket.py", line 748, in getaddrinfo
Nov 12 11:29:12 ravinder-adsb python3[23415]:     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
Nov 12 11:29:12 ravinder-adsb python3[23415]: socket.gaierror: [Errno -3] Temporary failure in name resolution
Nov 12 11:29:12 ravinder-adsb python3[23415]: During handling of the above exception, another exception occurred:
Nov 12 11:29:12 ravinder-adsb python3[23415]: Traceback (most recent call last):
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/local/bin/check_update.py", line 64, in <module>
Nov 12 11:29:12 ravinder-adsb python3[23415]:     schedule.run_pending()
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/local/lib/python3.7/dist-packages/schedule/__init__.py", line 780, in run_pending
Nov 12 11:29:12 ravinder-adsb python3[23415]:     default_scheduler.run_pending()
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/local/lib/python3.7/dist-packages/schedule/__init__.py", line 100, in run_pending
Nov 12 11:29:12 ravinder-adsb python3[23415]:     self._run_job(job)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/local/lib/python3.7/dist-packages/schedule/__init__.py", line 172, in _run_job
Nov 12 11:29:12 ravinder-adsb python3[23415]:     ret = job.run()
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/local/lib/python3.7/dist-packages/schedule/__init__.py", line 661, in run
Nov 12 11:29:12 ravinder-adsb python3[23415]:     ret = self.job_func()
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/local/bin/check_update.py", line 34, in update
Nov 12 11:29:12 ravinder-adsb python3[23415]:     raise e
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/local/bin/check_update.py", line 31, in update
Nov 12 11:29:12 ravinder-adsb python3[23415]:     f"https://{domain_server}/{check_version_path}")
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
Nov 12 11:29:12 ravinder-adsb python3[23415]:     return opener.open(url, data, timeout)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/urllib/request.py", line 525, in open
Nov 12 11:29:12 ravinder-adsb python3[23415]:     response = self._open(req, data)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/urllib/request.py", line 543, in _open
Nov 12 11:29:12 ravinder-adsb python3[23415]:     '_open', req)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
Nov 12 11:29:12 ravinder-adsb python3[23415]:     result = func(*args)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/urllib/request.py", line 1367, in https_open
Nov 12 11:29:12 ravinder-adsb python3[23415]:     context=self._context, check_hostname=self._check_hostname)
Nov 12 11:29:12 ravinder-adsb python3[23415]:   File "/usr/lib/python3.7/urllib/request.py", line 1326, in do_open
Nov 12 11:29:12 ravinder-adsb python3[23415]:     raise URLError(err)
Nov 12 11:29:12 ravinder-adsb python3[23415]: urllib.error.URLError: <urlopen error [Errno -3] Temporary failure in name resolution>
Nov 12 11:29:12 ravinder-adsb systemd[1]: ravinder-update.service: Main process exited, code=exited, status=1/FAILURE
Nov 12 11:29:12 ravinder-adsb systemd[1]: ravinder-update.service: Failed with result 'exit-code'.
Nov 12 11:29:13 ravinder-adsb frpc[670]: 2023/11/12 11:29:13 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:13 ravinder-adsb frpc[670]: 2023/11/12 11:29:13 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:14 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:29:14 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:29:14 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 381ms
Nov 12 11:29:14 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:29:15 ravinder-adsb systemd[1]: ravinder-update.service: Service RestartSec=2s expired, scheduling restart.
Nov 12 11:29:15 ravinder-adsb systemd[1]: ravinder-update.service: Scheduled restart job, restart counter is at 92.
Nov 12 11:29:15 ravinder-adsb systemd[1]: Stopped Ravinder Update.
Nov 12 11:29:15 ravinder-adsb systemd[1]: Started Ravinder Update.
Nov 12 11:29:15 ravinder-adsb frpc[670]: 2023/11/12 11:29:15 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:15 ravinder-adsb frpc[670]: 2023/11/12 11:29:15 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:17 ravinder-adsb frpc[670]: 2023/11/12 11:29:17 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:17 ravinder-adsb frpc[670]: 2023/11/12 11:29:17 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:19 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:29:19 ravinder-adsb frpc[670]: 2023/11/12 11:29:19 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:19 ravinder-adsb frpc[670]: 2023/11/12 11:29:19 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:21 ravinder-adsb frpc[670]: 2023/11/12 11:29:21 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:21 ravinder-adsb frpc[670]: 2023/11/12 11:29:21 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:23 ravinder-adsb frpc[670]: 2023/11/12 11:29:23 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:23 ravinder-adsb frpc[670]: 2023/11/12 11:29:23 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:25 ravinder-adsb frpc[670]: 2023/11/12 11:29:25 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:25 ravinder-adsb frpc[670]: 2023/11/12 11:29:25 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:27 ravinder-adsb frpc[670]: 2023/11/12 11:29:27 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:27 ravinder-adsb frpc[670]: 2023/11/12 11:29:27 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:29 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:29:29 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:29:29 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 383ms
Nov 12 11:29:29 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:29:29 ravinder-adsb frpc[670]: 2023/11/12 11:29:29 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:29 ravinder-adsb frpc[670]: 2023/11/12 11:29:29 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:31 ravinder-adsb frpc[670]: 2023/11/12 11:29:31 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:31 ravinder-adsb frpc[670]: 2023/11/12 11:29:31 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:33 ravinder-adsb frpc[670]: 2023/11/12 11:29:33 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:33 ravinder-adsb frpc[670]: 2023/11/12 11:29:33 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:34 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:29:35 ravinder-adsb frpc[670]: 2023/11/12 11:29:35 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:35 ravinder-adsb frpc[670]: 2023/11/12 11:29:35 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:37 ravinder-adsb frpc[670]: 2023/11/12 11:29:37 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:37 ravinder-adsb frpc[670]: 2023/11/12 11:29:37 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:39 ravinder-adsb frpc[670]: 2023/11/12 11:29:39 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:39 ravinder-adsb frpc[670]: 2023/11/12 11:29:39 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:41 ravinder-adsb frpc[670]: 2023/11/12 11:29:41 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:41 ravinder-adsb frpc[670]: 2023/11/12 11:29:41 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:43 ravinder-adsb frpc[670]: 2023/11/12 11:29:43 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:43 ravinder-adsb frpc[670]: 2023/11/12 11:29:43 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:44 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:29:44 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:29:44 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 385ms
Nov 12 11:29:44 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:29:45 ravinder-adsb frpc[670]: 2023/11/12 11:29:45 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:45 ravinder-adsb frpc[670]: 2023/11/12 11:29:45 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:47 ravinder-adsb frpc[670]: 2023/11/12 11:29:47 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:47 ravinder-adsb frpc[670]: 2023/11/12 11:29:47 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:49 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:29:49 ravinder-adsb frpc[670]: 2023/11/12 11:29:49 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:49 ravinder-adsb frpc[670]: 2023/11/12 11:29:49 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:51 ravinder-adsb frpc[670]: 2023/11/12 11:29:51 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:51 ravinder-adsb frpc[670]: 2023/11/12 11:29:51 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:53 ravinder-adsb frpc[670]: 2023/11/12 11:29:53 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:53 ravinder-adsb frpc[670]: 2023/11/12 11:29:53 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:55 ravinder-adsb frpc[670]: 2023/11/12 11:29:55 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:55 ravinder-adsb frpc[670]: 2023/11/12 11:29:55 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:57 ravinder-adsb frpc[670]: 2023/11/12 11:29:57 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:57 ravinder-adsb frpc[670]: 2023/11/12 11:29:57 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:29:59 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:29:59 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:29:59 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 386ms
Nov 12 11:29:59 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:29:59 ravinder-adsb frpc[670]: 2023/11/12 11:29:59 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:29:59 ravinder-adsb frpc[670]: 2023/11/12 11:29:59 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:01 ravinder-adsb frpc[670]: 2023/11/12 11:30:01 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:30:01 ravinder-adsb frpc[670]: 2023/11/12 11:30:01 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:03 ravinder-adsb frpc[670]: 2023/11/12 11:30:03 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:30:03 ravinder-adsb frpc[670]: 2023/11/12 11:30:03 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:04 ravinder-adsb reconnect-modem.py[471]: ping: Warning: source address might be selected on device other than wwan0.
Nov 12 11:30:05 ravinder-adsb frpc[670]: 2023/11/12 11:30:05 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:30:05 ravinder-adsb frpc[670]: 2023/11/12 11:30:05 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:07 ravinder-adsb frpc[670]: 2023/11/12 11:30:07 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:30:07 ravinder-adsb frpc[670]: 2023/11/12 11:30:07 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:09 ravinder-adsb frpc[670]: 2023/11/12 11:30:09 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:30:09 ravinder-adsb frpc[670]: 2023/11/12 11:30:09 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:11 ravinder-adsb frpc[670]: 2023/11/12 11:30:11 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:30:11 ravinder-adsb frpc[670]: 2023/11/12 11:30:11 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:13 ravinder-adsb frpc[670]: 2023/11/12 11:30:13 #033[1;34m[I] [service.go:177] [4fead55a7f82f4bc] try to reconnect to server...#033[0m
Nov 12 11:30:13 ravinder-adsb frpc[670]: 2023/11/12 11:30:13 #033[1;33m[W] [service.go:180] [4fead55a7f82f4bc] reconnect to server error: dial tcp 103.153.2.125:7000: connect: network is unreachable#033[0m
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: PING 8.8.8.8 (8.8.8.8) from 0.0.0.0 wwan0: 0(28) bytes of data.
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: --- 8.8.8.8 ping statistics ---
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 10 packets transmitted, 0 received, 100% packet loss, time 387ms
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: connect: Network is unreachable
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: 2
Nov 12 11:30:14 ravinder-adsb reconnect-modem.py[471]: (1, 2)
